### ISSUE TITLE
ReferenceError: window is not defined  at isLocalStorage 

### ISSUE THREAD

**Issue by janne-nygard-pinja** [klevultd/frontend-sdk#623]
I'm trying to run the klevu/core in a backend application. Seems that there is a dependency to the window in the isLocalStorage function. The rest of the code seems to indicate that this should run without browser so I'm thinking this might be a bug.

`ReferenceError: window is not defined
  at isLocalStorage (file:///workspaces/project/node_modules/@klevu/core/dist/mjs/utils/storage.js:12:49)
  at Function.KlevuStorage.getItem (file:///workspaces/project/node_modules/@klevu/core/dist/mjs/utils/storage.js:36:9)
  at new KlevuUserSession (file:///workspaces/project/node_modules/@klevu/core/dist/mjs/usersession.js:47:36)
  at Function.init (file:///workspaces/project/node_modules/@klevu/core/dist/mjs/usersession.js:53:36)
  at Function.init (file:///workspaces/project/node_modules/@klevu/core/dist/mjs/config.js:50:26)
  at Object.productSearchResolver [as productSearch] (file:///workspaces/project/packages/project-backend-dummy/src/resolvers/catalog/productSearchResolver.ts:27:17)
  at field.resolve (file:///workspaces/project/node_modules/@apollo/server/src/utils/schemaInstrumentation.ts:82:22)
  at executeField (/workspaces/project/node_modules/graphql/execution/execute.js:492:20)
  at executeFields (/workspaces/project/node_modules/graphql/execution/execute.js:414:22)
  at executeOperation (/workspaces/project/node_modules/graphql/execution/execute.js:344:14
`

A possible fix could be to add the isBrowser check to the isLocalStorage -function. (The isSessionStorage might have the same issue. Not getting a error from it though.)

```
import { isBrowser } from "./isBrowser.js";

...

const isLocalStorage = (storageType) => {
    return storageType === StorageType.LOCAL && isBrowser() && window.localStorage;
};

```

**Comment by janne-nygard-pinja** [comment]
The code used from the klevu/core

```

   import { KlevuFetch, search } from '@klevu/core';

    KlevuConfig.init({
        url: 'https://XXXXXXX.ksearchnet.com/cs/v2/search',
        apiKey: 'klevu-XXXXXXXX',
    });

    const result = await KlevuFetch(search('test'));
```

**Comment by nikhil-narayana-klevu** [comment]
@janne-nygard-pinja this issue has been fixed in version 5.2.1. let me know if it doesnt work for you.
